web/api/mocks/data/steps.json

```diff
@@ -1235,14 +1226,11 @@
     "targetDatabase": "staging",
     "isValid": false,
     "options": {
-      "input_file_path": ".",
-      "input_file_type": "documents",
+      "inputFilePath": ".",
+      "inputFileType": "documents",
       "output_collections": "Order,Flow 4 Ingest Step,input,newCollection",
-      "output_permissions": "rest-reader,read,rest-writer,update",
-      "document_type": "json",
-      "transform_module": "/data-hub/5/transforms/mlcp-flow-transform.sjs",
-      "transform_namespace": "http://marklogic.com/data-hub/mlcp-flow-transform",
-      "transform_param": "entity-name=Order,flow-name=Flow 4 Ingest Step"
+      "outputPermissions": "rest-reader,read,rest-writer,update",
+      "outputFileType": "json"
```

Comment:
To avoid conflicts I suggest rollback ingest options. My PR will take care of that.